llarp/handlers/tun.cpp

```diff
@@ -32,6 +32,13 @@ namespace llarp
       m_NetworkToUserPktQueue.Process(send);
     }
 
+    bool
+    TunEndpoint::ShouldFlushNow(llarp_time_t now) const
+    {
+      static constexpr llarp_time_t FlushInterval = 25;
```

Comment:
I wonder if this (and other constants in other files) should be at the top of the class/file so all constants for a given class are in one place.  Tough call though.